Dockerfile

```diff
@@ -47,9 +51,9 @@ LABEL cilium-sha=${CILIUM_SHA}
 LABEL maintainer="maintainer@cilium.io"
 COPY --from=builder /tmp/install /
 COPY --from=cilium-envoy / /
-COPY plugins/cilium-cni/cni-install.sh /cni-install.sh
-COPY plugins/cilium-cni/cni-uninstall.sh /cni-uninstall.sh
-COPY contrib/packaging/docker/init-container.sh /init-container.sh
+COPY --from=builder /go/src/github.com/cilium/cilium/plugins/cilium-cni/cni-install.sh /cni-install.sh
+COPY --from=builder /go/src/github.com/cilium/cilium/plugins/cilium-cni/cni-uninstall.sh /cni-uninstall.sh
+COPY --from=builder /go/src/github.com/cilium/cilium/contrib/packaging/docker/init-container.sh /init-container.sh
```

Comment:
How do we make sure these files are the actual versions from the latest tree exactly? Are we relying on the builder image always being rebuilt now?